foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,46 @@
+    *   Make sure your `playerSelection` parameter is case-insensitive so that users can input `rock`, `ROCK`, `RocK`, or any other variation.
+   -  **Important note:** you want to `return` the results of `playRound` call, *not* `console.log` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
+
+      ~~~javascript
+      function playRound(playerSelection, computerSelection) {
+        // your code here!
+      }
  
-    const playerSelection = "rock";
-    const computerSelection = getComputerChoice();
-    console.log(playRound(playerSelection, computerSelection));
-    ~~~
-
-6.  Write a NEW function called `game()`. Use the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
-    *   You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
-    *   At this point you should be using `console.log()` to display the results of each round and the winner at the end.
-    *   Use `prompt()` to get input from the user. [Read the docs here if you need to.](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
+      const playerSelection = "rock";
+      const computerSelection = getComputerChoice();
+      console.log(playRound(playerSelection, computerSelection));
+      ~~~
+
+5. Write a new function called `game` that plays a 5 round game, keeps the game's score, and reports a winner or loser at the end. Use the previous function *inside* this `game` function.
+    *   You have not officially learned how to "loop" over code to repeat function calls... If you already know about loops from somewhere else or feel like learning more, feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. In the next lesson, we will cover loops.
+    *   At this point, you should be using `console.log` to display the results of each round and the winner at the end.
+    *   Use the `prompt` function to get input from the user. [Read the docs here if you need to.](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
     *   Feel free to re-work your previous functions if you need to. Specifically, you might want to change the return value to something more useful.
-    *   Feel free to create more "helper" functions if you think it would be useful.
+    *   Feel free to create more "helper" functions if you think they would be useful.
```

Comment:
- `-` for list items per style guide 
- perhaps a more descriptive links for the MDN documentation about `prompt`

Reply:
Maybe?
>  To obtain the choice from the human user, use the `prompt` function. You can refer to the [official MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) to learn how to use the prompt function and its parameters.

Reply:
I think we should also add the chapter :) 
> To obtain the choice from the human player, use the prompt method. You can refer to the [prompt() method documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) to learn how to use the prompt method and its parameters.

Feel free to simplify the last part after the link to your liking. It's probably too much prompt repetition.